warn/warn_bazel_api.go

```diff
@@ -798,6 +798,27 @@ func nativeProtoSymbolsWarning(symbol string, bzlfile string) func(f *build.File
 	}
 }
 
+func nativeShBinaryWarning(f *build.File, fileReader *FileReader) []*LinterFinding {
+	if f.Type != build.TypeBzl && f.Type != build.TypeBuild {
+		return nil
+	}
+	return NotLoadedFunctionUsageCheck(f, fileReader, []string{"sh_binary"}, "@rules_shell//shell:sh_binary.bzl")
```

Comment:
Can you move hard coded  "@rules_shell//shell:" into tables. See other warnings.

This makes it a bit easier to import, because we're overriding those labels internally.

Reply:
ah nice, i skipped that not knowing you were replacing that internally, i should have known! Updated!